Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Tests/CustomImageComparatorTest.cs

```diff
@@ -10,15 +10,19 @@ public class CustomImageComparatorTest : BaseTest
         [Test(Description = "TC-0005 Check Custom Image Comparator")]
         public void TC0005_CheckCustomImageComparator()
```

Comment:
we should not say that we're checking the comparator, it is a utilitity for testing, not an object of testing, cause we're not writing a unit test here. Better to say in the name that we're comparing two images

Reply:
Code was updated.